### PR TITLE
docs: add access monitoring notice for external audit storage

### PR DESCRIPTION
**Pull request by stevenGravy**
We've seen some issues raised where folks are seeing that External Audit Storage does not allow to use Access Monitoring in Teleport Cloud.  Adding this notice until it is supported.

### CODE REVIEW

File: docs/pages/access-controls/access-monitoring.mdx [hunk 1]
```diff
@@ -15,6 +15,12 @@ or database sessions without MFA) and provides users with recommendations on sug
 
 Users are able to write their own custom access monitoring queries by querying the audit log.
 
+<Admonition type="notice">
+  Access Monitoring is not currently supported with External Audit Storage
+  in Teleport Enterprise (cloud-hosted). A upcoming version will support Access Monitoring in
+  that configuration.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:21]:
```suggestion
  in Teleport Enterprise (cloud-hosted). This functionality will be
  enabled in a future Teleport release.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 2]
```diff
@@ -298,7 +304,7 @@ WHERE
 
 ### Query Examples with Assist AI
 
-The query editor also integrates with Teleport Assist, making it easy to generate queries with a human-readable description of the data you want to retrieve.
+The query editor also integrates with Teleport Assist, allowing to generate queries with a human-readable description of the data you want to retrieve.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:307]: This sounded better before IMO. I don't think _allowing to_ is grammatically correct. (allowing _who_ to?)

Comment (stevenGravy) [reply docs/pages/access-controls/access-monitoring.mdx:307]: "easy" is tagged as a error now in the linting why I changed it. Will look for some other options. Any suggestions @ptgott ?

Comment (ptgott) [reply docs/pages/access-controls/access-monitoring.mdx:307]: I think "allowing you to" is fine, as the sentence already indicates that the data is something the reader wants to retrieve.

Comment (stevenGravy) [reply docs/pages/access-controls/access-monitoring.mdx]: Please see verbiage update

**Review by nklaassen — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Pull request merged**